lld/test/ELF/armv8-thumb-plt-reloc.s

```diff
@@ -0,0 +1,121 @@
+// REQUIRES: arm
```

Comment:
Can you add a big-endian test to make sure we've got the 4-byte Thumb2 instruction reversal right?